.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
```

Comment:
Where do you plan to include a check for PR description and commit message?

Reply:
I'm not sure what you mean by a check? An automated check?

Reply:
I meant a checkbox. But any other solution is also fine - as a reminder.

Reply:
I'm not clear what a reminder for commit messages might say. I thought the commit message linting was going to be checked by some kind of automation?

Reply:
For PR title and commits, how about:
```
Please give the pull request a short but descriptive title.
Use [conventional commits](https://www.conventionalcommits.org/) to separate and summarize commits logically.
```
@nasthagiri you said something today about not using Jira issue numbers in the title.  Some of our existing automation relies on this, and people find the short-hand useful, so I'm not sure we should push people away from that habit.

Reply:
I was under the impression that our existing automation would look at the JIRA numbers in the PR description in addition to the title.  If there's a risk of breaking those, then ok, we can add that later.

Reply:
+1 to the proposed change for the descriptive title and the conventional commits.